scripts/fast-reboot

```diff
@@ -16,6 +16,7 @@ ASSISTANT_SCRIPT="/usr/bin/neighbor_advertiser"
 DEVPATH="/usr/share/sonic/device"
 PLATFORM=$(sonic-cfggen -H -v DEVICE_METADATA.localhost.platform)
 PLATFORM_PLUGIN="${REBOOT_TYPE}_plugin"
+PLATFORM_LOG_SSD_HEALTH="platform_log_ssd_health"
```

Comment:
as I suggested in https://github.com/Azure/sonic-buildimage/pull/4214, can we make the platform_log_ssd_health as a common utility to check the ssd status (maybe we can do specifically for innodisk's "3IE3" type or generically for all ssd devices)?

Reply:
Sujin,
As suggested, I have made log_ssd_health as generic and it has been called with ExecStartPre in fstrim.service. Removed the check for 3IE3 and now it will collect smartctl output before fast-reboot and fstrim service start in all platforms. I have verified the fix in few Dell Platforms(S6100 and Z9264F)